EBNF.Compiler/Compiler.cs

```diff
@@ -310,6 +345,7 @@ public static CompilerResults CreateDll(string[] code, string filePath, string k
 
             parameters.CompilerOptions =
                 $"/doc:\"{Path.GetDirectoryName(Path.GetFullPath(filePath))}\\{Path.GetFileNameWithoutExtension(filePath)}.xml\" /optimize";
+            parameters.ReferencedAssemblies.Add(typeof(Regex).Assembly.FullName);
```

Comment:
I'm fairly sure this would have also worked, but been clearer:
```suggestion
            parameters.ReferencedAssemblies.Add("System.Text.RegularExpressions.dll"); // For RegEx classes
```

Please test whether this works though first.